mkdocs.yml.tmpl

```diff
@@ -0,0 +1,155 @@
+site_name: Kubernetes Gateway API
```

Comment:
Seems like we need a comment in the generated file that instructs users to exclusively update the template file instead + the make command required to update the file.

Reply:
Consider using MkDocs [Configuration Inheritance](https://www.mkdocs.org/user-guide/configuration/#configuration-inheritance) to separate the base configuration from the nav config.

Reply:
On second thought, since this is just modifying the navigation, consider implementing this as a hook/plugin that runs when the [`on_nav`](https://www.mkdocs.org/dev-guide/plugins/#on_nav) event is called.